crates/iota-genesis-builder/src/stardust/types/address_swap_map.rs

```diff
@@ -182,7 +182,7 @@ fn verify_headers(headers: &csv::StringRecord) -> Result<(), anyhow::Error> {
     const LEFT_HEADER: &str = "Origin";
     const RIGHT_HEADER: &str = "Destination";
 
-    if &headers[0] != LEFT_HEADER && &headers[1] != RIGHT_HEADER {
+    if &headers[0] != LEFT_HEADER || &headers[1] != RIGHT_HEADER {
         anyhow::bail!("Invalid CSV headers");
     }
```

Comment:
Something like this could be even nicer I think:
```suggestion
    anyhow::ensure!(
        headers.len() == 2
            && &headers[0] == "Origin"
            && &headers[1] == "Destination",
        "Invalid CSV headers"
    );
```
(and remove the const above)

Reply:
(I've seen later that this is the way we used for the address swap split map)

Reply:
Done